### DISCUSSION TITLE
Moved osu directory, i messed up the osu:// link handler

### DISCUSSION THREAD

**Discussion by Kaloyanes** [ppy/osu#16073] [Q&A] [answered]
The title says it all. Just an idiot who didn't think this through. So is there a way to fix this without reinstalling osu because i have 10,000 beatmaps.

**Comment by bdach** [comment, marked as answer]
are you asking about stable or lazer? if i recall correctly lazer doesn't even handle `osu://` links.

**Reply by Kaloyanes** [reply]
Stable

**Reply by bdach** [reply]
this repository is for lazer things only. but i'm feeling like making an exception today: i believe your issue will be fixed if you click the "repair folder permissions" button in settings.

**Reply by Kaloyanes** [reply]
This worked. Thanks! And sorry that I typed into this repo.
